src/guides/v2.3/javascript-dev-guide/widgets/widget_gallery.md

```diff
@@ -580,3 +580,56 @@ api.updateOptions([{
 [Fotorama widget]: http://fotorama.io/
 [lib/web/mage/gallery/gallery.js]: {{ site.mage2bloburl }}/{{ page.guide_version }}/lib/web/mage/gallery/gallery.js
 [lib/web/magnifier/magnify.js]: {{ site.mage2bloburl }}/{{ page.guide_version }}/lib/web/magnifier/magnify.js
+
+## Code sample
+
+This example shows a use case for a gallery widget on any page.
+
+```html
+<div class="image-gallery"></div>
+ 
+<script>
+require ([
+    'jquery',
+    'mage/gallery/gallery'
+], function ($, gallery) {
+    $(function () {
+        $('.image-gallery').each(function (index, element) {
+            gallery({
+                options: {
+                    "nav": "false",
+                    "loop": "true",
+                    "arrows": "true"
+                },
+                data: [
+                    { img: "<image_url_1>" },
+                    { img: "<image_url_2>" },
+                    { img: "<image_url_3>" }
+                ],
+                fullscreen: {
+                    "navdir": "horizontal"
+                },
+                "breakpoints": {
+                    "<breakpoint_name>": {
+                        "conditions": {
+                            "max-width": "767px"
+                        },
+                        "options": {}
+                    }
+                }
+            }, element);
+        });
+    });
+});
+</script>
+```
+
+The breakpoints options are set in the `view.xml` configuration file of a theme. The file is conventionally located in `<theme_dir>/etc`.
```

Comment:
```suggestion
The breakpoints options are set in the `view.xml` configuration file of a theme. The file is located in `<theme_dir>/etc`.
```

Reply:
Hi @BarnyShergold 
All your suggestions have been implemented!

Could you please verify this PR?

Thank you!